docs/notebooks/test_snia.ipynb

```diff
@@ -96,7 +96,8 @@
     "res, passbands = run_snia_end2end(\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    plt.plot(z2, nsn * 0.0005)
</pre><p>What does this 0.0005 mean?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lincc-frameworks/tdastro/pull/175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Random number again to look at the distributions in the same scale (me being lazy not renormalizing any of those). I need to change this to use the actual volumetric_rate*dV function instead of the integrated one, but not for this PR. With larger number of bins the function is close enough for testing purpose right now.</p>